docs/pages/components/select.md

```diff
@@ -25,7 +25,7 @@ For lists that require more than 12 options, the <a href="/patterns/combobox-inp
      <div class="fd-select">
          <div class="fd-select__control" role="button" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
              Select
-             <span class="fd-button fd-button--transparent sap-icon--slim-arrow-down fd-select__button"></span>
```

Comment:
This was changed to a `span` to fix the duplicate tabbing issue (you can observe this now by tabbing into the Select component in this PR's deployment).  I guess you could make an argument that anything with the `fd-button` class should be an actual `button`, but the parent being `role="button"` is sufficient for screen readers and keyboard users to use the component without majorly refactoring the styles.